libs/common-services/src/test/java/uk/gov/di/ipv/core/library/helpers/VerifiableCredentialGenerator.java

```diff
@@ -49,7 +52,63 @@ public static String generateVerifiableCredential(
                         .claim(EXPIRATION_TIME, now.plusSeconds(600).getEpochSecond())
                         .claim(VC_CLAIM, vcClaim)
                         .build();
+        return signTestJWT(claimsSet);
+    }
+
+    public static String generateVerifiableCredential(
+            TestVc vcClaim, String subject, String issuer, Instant nbf) throws Exception {
+        JWTClaimsSet claimsSet =
+                new JWTClaimsSet.Builder()
+                        .claim(SUBJECT, subject)
+                        .claim(ISSUER, issuer)
+                        .claim(NOT_BEFORE, nbf.getEpochSecond())
+                        .claim(VC_CLAIM, vcClaim)
+                        .build();
+        return signTestJWT(claimsSet);
+    }
+
+    public static String generateVerifiableCredential(TestVc vcClaim) throws Exception {
+        Instant now = Instant.now();
+        JWTClaimsSet claimsSet =
+                new JWTClaimsSet.Builder()
+                        .claim(SUBJECT, "urn:uuid:811cefe0-7db6-48ad-ad89-0b93d2259980")
+                        .claim(ISSUER, "https://review-p.staging.account.gov.uk")
+                        .claim(NOT_BEFORE, now.getEpochSecond())
+                        .claim(VC_CLAIM, vcClaim)
+                        .build();
+        return signTestJWT(claimsSet);
+    }
+
+    public static String generateVerifiableCredential(
```

Comment:
Could the other overloads of this method delegate to this method - the most flexible one?

Reply:
So I was thinking the method on line 70 would become:
```
    public static String generateVerifiableCredential(TestVc vcClaim) throws Exception {
        return generateVerifiableCredential(
                vcClaim,
                "urn:uuid:811cefe0-7db6-48ad-ad89-0b93d2259980",
                "https://review-p.staging.account.gov.uk",
                null,
                null,
                null);
    }
```